build/task-build-deps.js

```diff
@@ -43,19 +78,33 @@ async function findNativeModules() {
 async function rebuild() {
   console.log('Rebuilding modules...');
   const command = path.join(__dirname, '..', 'node_modules', '.bin', 'electron-rebuild');
-  await BuildUtils.spawn(command, ['-f'], {
+  await BuildUtils.spawn(command, [
+    '-f',
+    '-e', BuildUtils.getElectronRoot(),
+    '-v', BuildUtils.getElectronVersion(),
+  ], {
     stdio: 'inherit',
   });
 }
 
 export default async function() {
   let existingConfig = {};
   try {
-    existingConfig = await fs.readJson(BUILD_CONFIG_JSON);
+    existingConfig = await fs.readJson(BuildUtils.getBuildConfigFile());
   } catch (e) {
     // Missing files mean we rebuild
   }
 
+  const electron = BuildUtils.getManifest()._electron;
+  try {
+    if (electron.version !== BuildUtils.getElectronVersion()) {
```

Comment:
Let's not do this, or at least not outside of automation.  There's no sense pulling the same large ZIP file twice in order to discover there's something funky with the compression or whatever.